src/DynamoCore/Configuration/IPathResolver.cs

```diff
@@ -164,9 +164,9 @@ public interface IPathManager
         string PythonTemplateFilePath { get; }
 
         /// <summary>
-        /// Full path to the GalleryContent xml file. The file is located in
-        /// the AppData/Dynamo/[version]/gallery/
+        /// Full path to the legacy GalleryContent xml file.
         /// </summary>
+        [System.Obsolete("This property will be removed in Dynamo 3.0")]
```

Comment:
@mjkkirschner If I remember correctly, we cant remove public properties in DynamoCore right? So adding Obsolete property, but I removed view layer commands and functions